Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/SettingsItem.java

```diff
@@ -38,6 +39,18 @@
    * @param nameId        Resource ID for a text string to be displayed as this setting's name.
    * @param descriptionId Resource ID for a text string to be displayed as this setting's description.
    */
+  public SettingsItem(String key, String section, Setting setting, int nameId, int descriptionId,
```

Comment:
Is there a better way to handle `gameId` instead of spread it every where?

Reply:
Setting object needs the gameId for when the button binding is caught, otherwise it will not know what game to save the binding for(or that it is for a game at all).

https://github.com/dolphin-emu/dolphin/blob/be0fc8e7d5c327cb65520c930bbfe0c072645f5d/Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/MotionAlertDialog.java#L215